GAMEFLOW.md

```diff
@@ -0,0 +1,1306 @@
+  <tr valign="top">
+    <td>
+      <code>key3</code>
+    </td>
+    <td>Describes pickup object ID 131 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>key4</code>
+    </td>
+    <td>Describes pickup object ID 132 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>pickup1</code>
+    </td>
+    <td>Describes pickup object ID 141 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>pickup2</code>
+    </td>
+    <td>Describes pickup object ID 142 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>puzzle1</code>
+    </td>
+    <td>Describes pickup object ID 110 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>puzzle2</code>
+    </td>
+    <td>Describes pickup object ID 111 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>puzzle3</code>
+    </td>
+    <td>Describes pickup object ID 112 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>puzzle4</code>
+    </td>
+    <td>Describes pickup object ID 113 in the inventory top-ring.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>title</code>
+    </td>
+    <td>String</td>
+    <td>Yes</td>
+    <td colspan="2">The title of the level.</td>
+  </tr>  
+  <tr valign="top">
+    <td rowspan="8">
+      <code>type</code>
+    </td>
+    <td rowspan="8">String</td>
+    <td rowspan="8">Yes</td>
+    <td colspan="2">
+      The level type, which must be one of the following values.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <strong>Type</strong>
+    </td>
+    <td>
+      <strong>Description</strong>
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>bonus</code>
+    </td>
+    <td>
+      Only playable when all secrets are collected. See
+      <a href="#bonus-levels">Bonus levels</a> for full details.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>current</code>
+    </td>
+    <td>
+      One level of this type is necessary to read TombATI's save files. OG has a
+      special level called <code>LV_CURRENT</code> to handle save/load logic.
+      TR1X does away with this hack. However, the existing save games expect the
+      level count to match, otherwise the game will crash.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>cutscene</code>
+    </td>
+    <td>A cutscene level.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>gym</code>
+    </td>
+    <td>
+      At most one of these can be defined. Accessed from the photo option
+      (object ID 73) on the title screen. If omitted, the photo option is not
+      displayed.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>normal</code>
+    </td>
+    <td>A standard level.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>title</code>
+    </td>
+    <td>One - and only one - of these must be defined.</td>
```

Comment:
Maybe add `The main menu level. One -...` as a preface? Just trying to avoid any confusion since normal levels also have a level `title` which is the name of the level.

Reply:
Good catch - done.